### ISSUE TITLE
Dialog Window a piece of the lower image area is always cut off

### ISSUE THREAD

**Issue by andre-bu** [ioBroker/ioBroker.vis-jqui-mfd#28]
Hello, in my dialog window, a piece of the lower image area is always cut off. If I then increase the dialog height a little, the image is distorted, but the same area is still missing. I can then enlarge the window in the VIS by dragging it down and then the missing area is also visible.

**Comment by Apollon77** [comment]
I think in this case the words "n image tells more then 1000 words" is true :-) Can you please add an image that shows the issue?

**Comment by andre-bu** [comment]
@Apollon77 
You're right, I made a short video. That's the best way to see the problem.
[https://youtu.be/bEspKvnj0RM](url)
